_posts/2015-05-27-interesting-work-not-interesting-jobs.md

```diff
@@ -0,0 +1,40 @@
+
+In this context _value_ means companies where your work will have a significant impact on the business. Software has a huge impact at Google, for example, but if you're the 20,000th sotware engineer how much marginal impact are you going to have? If, however, you are the 20th engineer at a mature business with money in its pockets and a leadership interested in innovation, it's likely that you can play a large part in whatever technology they decide to develop. You'll probably get a chance to help make those decisions.
+
+Here are a couple of examples of companies that, a few years ago at least, would not have been obvious candidates for “interesting” jobs:
+
+* New York Times: An old media stalwart, founded in 1851 it produces some [incredible data visualizations][nytime-data-viz]
+* Etsy: a marketplace for buying and selling unique goods. Built the popular [statsd library][statsd] and popularised the idea of ['Measure Anything, Measure Everything'][etsy-blog]
+
+Or take, for example, just plucked at random, Stitch Fix, where I work. Stitch Fix is an online styling service that makes its money selling clothes to women. Since Stitch Fix is an innovator in the retail industry people already in the retail industry are pretty excited to work here. That's why the [executive team][exec-team] of this still young startup is made up of people whose former jobs were COO at Walmart.com, a GM at Nike, CMO at Sephora, and VP Data Science at Netflix.
+
+Of course, not many software professionals are desperate to work in the retail industry, but consider the problems we are trying to solve (to name just a few):
+
+* Stitch Fix has built a unique styling platform used by over 1,500 stylists who work 10 to 30 hrs/week. This platform, designed and built by the engineering team, allows stylists to work with a short list of styles, recommended by an algorithm specifically for each client, to pick five items the customer will love. How would you build a tool that can handle a different data set for every client as well as presenting all a client's previous shopping history, in a way that improves a stylist's efficiency and success rate?
+* Our customers give us enormous amount of data about themselves.  We also have a lot of data about  the clothes we sell. Our world-class [data science team][data-sci-team] produces insightful analysis about the interactions between the client and clothes. How can you  integrate real-time sales data and client feedback analysis into the tools where the buying teams are planning and placing orders for inventory six months from now?
```

Comment:
amount should be amounts, or 'give us an enormous amount'

there's an extra space after 'data about __ the clothes', and 'How can you __ integrate'

'produces insightful analysis' could have 'analyses' instead